bridgetown-core/lib/bridgetown-core/front_matter/loaders.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+module Bridgetown
+  module FrontMatter
+    module Loaders
+      autoload :Base, "bridgetown-core/front_matter/loaders/base"
+      autoload :Ruby, "bridgetown-core/front_matter/loaders/ruby"
+      autoload :YAML, "bridgetown-core/front_matter/loaders/yaml"
+
+      Result = Struct.new(:content, :front_matter, :line_count, keyword_init: true)
```

Comment:
**comment:** This should likely be marked private. I believe it should be a temporary thing.